src/schema/v2/location.ts

```diff
@@ -119,9 +123,34 @@ export const LocationType = new GraphQLObjectType<any, ResolverContext>({
   }),
 })
 
-const Location: GraphQLFieldConfig<void, ResolverContext> = {
+export const Location: GraphQLFieldConfig<void, ResolverContext> = {
```

Comment:
Rename this to `LocationField`, we started calling these what they are and the same applies to types being called e.g. `LocationType`.

Reply:
👍